AI Game Jam/Assets/Scripts/PlayerMovement.cs

```diff
@@ -1,33 +1,52 @@
+/*
+* Description: Manages the Player Movement, uses the built in character controller component in unity
+* Author: Chase Bennett-Hill
+* Last Modified: 7 / 03 / 24
+* Last Modified By: Chase Bennett-Hill
+*/
 using System.Collections;
 using System.Collections.Generic;
 using UnityEngine;
 
 public class PlayerMovement : MonoBehaviour
 {
-    private CharacterController controller;
-    private Vector3 playerVelocity;
-    private bool groundedPlayer;
-    [SerializeField] private float playerSpeed = 4.0f;
+
+    private CharacterController controller; //Character controller component
+    private Vector3 playerVelocity; //Player velocity value based on x y and z axis
+    [SerializeField] private bool groundedPlayer; //Checks if the player is touching the ground
+    [SerializeField] private float playerSpeed = 4.0f; //Player speed
+    [SerializeField] private float jumpHeight = 1.0f; //Jump height
```

Comment:
I remember you saying about being able to jump on 2^3 cubes but I couldn't do it, so I changed it to 1.5. We can always change this later!

Reply:
Yeah I was mistaken about that because I think my cubes were half in the ground.  I will have a play around with the values!